src/gov/nasa/worldwind/render/SurfaceText.java

```diff
@@ -153,6 +155,23 @@ public void setPosition(Position position)
         this.onShapeChanged();
     }
 
+    /** {@inheritDoc} */
+    public double getRotationAngle()
```

Comment:
The method-name would probably need to change `getHeading()` if the field is changed to `heading`. The return-type should then match the field-type (i.e. `Angle`).